install/bootstrap_livecd_usb.md

```diff
@@ -733,9 +731,9 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
    The following assumes the CSM_PATH environment variable is set to the absolute path of the unpacked CSM release.
 
    ```bash
-   pit:/var/www/ephemeral# rpm -Uvh --force $(find ./${CSM_RELEASE}/rpm/ -name "goss-servers*.rpm" | sort -V | tail -1)
-   pit:/var/www/ephemeral# rpm -Uvh --force $(find ./${CSM_RELEASE}/rpm/ -name "csm-testing*.rpm" | sort -V | tail -1)
-   pit:/var/www/ephemeral# cd 
+   pit:/var/www/ephemeral# rpm -Uvh --force $(find ${CSM_PATH}/rpm/cray/csm/ -name "goss-servers*.rpm" | sort -V | tail -1)
```

Comment:
I think we purposely moved these up a couple of directories to /rpm/ because these rpms kept moving around.

Reply:
Nice catch, this was from my copy-paste-and-fixup from working in both pages (usb<->remote)